scripts/gen_seed_test_data.py

```diff
@@ -75,7 +75,7 @@ def _generate_random_pii_record(faker):
                 use=random.choice(["home", "work", "mobile"]),
             )
         ],
-        race=random.choice(list(Race)),
+        race=str(random.choice(list(Race))),
```

Comment:
Right now, the `race` field accepts a list of race strings and not a single one.  We made this change a while back as multiple race values are supported in FHIR.  I'm interested though in what behavior you observed before making this change?

https://github.com/CDCgov/RecordLinker/blob/main/src/recordlinker/schemas/pii.py#L366

Reply:
Ah, definitely should have included the error in the PR:
```
pii.py", line 411, in parse_race
    return [Race.parse(v) for v in value]
                                   ^^^^^
TypeError: 'Race' object is not iterable
```

Reply:
Are you seeing any errors when you run the script?

Reply:
When I run the change, it shows up as a list of OTHER values.  For example

```json
"race": [
  "OTHER",
  "OTHER",
  "OTHER",
  "OTHER",
  "OTHER",
]
```

I think we might need to wrap it in a list.  Otherwise line 411 in pii.py is just iterating over the string and processing each character as a race value, all of which get categorized as OTHER.